openlibrary/core/vendors.py

```diff
@@ -187,6 +206,16 @@ def clean_amazon_metadata_for_load(metadata):
     if metadata.get('source_records'):
         asin = metadata.get('source_records')[0].replace('amazon:', '')
         conforming_metadata['identifiers'] = {'amazon': [asin]}
+    title, subtitle = split_amazon_title(metadata['title'])
+    conforming_metadata['title'] == title
+    if subtitle:
+        conforming_metadata['full_title'] = title + ' : ' + subtitle
```

Comment:
```suggestion
        conforming_metadata['full_title'] = title + ': ' + subtitle
```

Reply:
This is a funny one as I added the space deliberately to match what I thought was being done in the MARC import case, I'd need to confirm it, but I thought there was cataloguing practice that did things this way. I does have implications for detecting whether the title was modified in one of the comment areas further up, and I want to avoid saving title : subtitle in the notes field if there were no other changes. I'll check to see if there is a compelling reason to drop the extra space, and make sure we are consistent with it.

Reply:
I don't feel strongly about this; if the user never sees this text, feel free to resolve this as "Won't fix" :)

Reply:
I _was_ thinking about a real thing -- ISBD cataloguing practice and punctuation https://www.ifla.org/files/assets/cataloguing/isbd/isbd-cons_20110321.pdf which is / influences the punctuation used in MARC records. I've set a new constant `ISBD_UNIT_PUNCT` to ` : ` to make it look less like a typo, and have used it in a couple of places :)